week_02/backjoon/jclim/BOJ_1912.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <algorithm>
+
+using namespace std;
+
+int n;
+int dp[100001];
+int answer = -1001;
```

Comment:
```suggestion
int MAX_COUNT = 1000001;
int MIN_NUMBER = -1001;

int dp[MAX_N];
int answer = MIN_NUMBER;
```

c++을 내가 잘 몰라서 위와 같이 쓴게 문법적으로 맞는지 모르곘는데, 그냥 100001 , -1001 같은 매직넘버를 쓰는 것보다는
상수로 값을 extract 해주고 이름을 붙여주는 방법은 어떨까?? 😄

Reply:
지현이 코드 보고 왔는데, 상수 선언은 `#define` 쓰는거 같은데 그렇게 적어주면 좋을 것 같애~